### ISSUE TITLE
Handle -1 value for memory_limit

### ISSUE THREAD

**Issue by prudloff-insite** [MirazMac/php-requirements-checker#2]
We are checking that PHP has enough memory like this:
```php
$checker = new Checker();
$checker->requireIniValues(['memory_limit' => '>=512M']);
```

This works well for positive values, but `memory_limit` can also have a `-1` value (which means unlimited) and php-requirements-checker does not seem to handle that:
> The php.ini setting `memory_limit` should be greater than or equal to 512M.  Currently it's set to -1

**Comment by MirazMac** [comment]
Interesting. So, the fix would be to skip the check if an INI value is set to -1. I can do that but I need to make sure it won't produce any side effects for other values. Thanks for notifying tho.

**Comment by MirazMac** [comment]
This is now fixed in ``0.2`` (tested). Please update and it should work fine, thank you.

**Comment by prudloff-insite** [comment]
That was fast! I tried 0.2 and it does not complain about -1 anymore.
